pkg/webhook/sidecarhandler.go

```diff
@@ -14,6 +14,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+// SideCar Kubernetes Sidecar Injector schema
 type SideCar struct {
```

Comment:
We should probably name this `Sidecar` instead of `SideCar`, since we're naming other structs `SidecarInjectorPatcher` and not `SideCarInjectorPatcher`